### ISSUE TITLE
I can receive mail but i can't send them

### ISSUE THREAD

**Issue by BenaddiRar** [Mailu/Mailu#1327]
hello everyone and congratulation for this project.

so my issue is that i can receive mail from my mail account (r.benaddi@e-solutions.ma) to my mailu server but i can't send them.

i have do the instalaltion from this video tutorial : https://www.youtube.com/watch?v=ScarlmgD0dU

My server is a Droplet from digitalocean, with ubuntu 18.

a domain name configured with a cloudflare acount.

my docker-compose logs smtp is :

smtp_1       | Jan 26 19:48:48 mail postfix/cleanup[19382]: 5AB6AFE10F: replace: header Received: from mail.e-solution.cloud (mailu_webmail_1.mailu_default [192.168.203.11])??by mail.e-solution.cloud (Postfix) with ESMTPA id 5AB6AFE10F??for <r.benaddi@e-solutions.ma>; Sun, 26 Jan 2020 19 from mailu_webmail_1.mailu_default[192.168.203.11]; from=<admin@e-solution.cloud> to=<r.benaddi@e-solutions.ma> proto=ESMTP helo=<mail.e-solution.cloud>: Received: from authenticated-user (PRIMARY_HOSTNAME [PUBLIC_IP])??by mail.e-solution.cloud (Postfix) with ESMTPA id 5AB6AFE10F??for <r.benaddi@e-solutions.ma>; Sun, 26 Jan 2020 19:48:48 +0000 (UTC)
smtp_1       | Jan 26 19:49:19 mail postfix/smtp[19383]: 5AB6AFE10F: to=<r.benaddi@e-solutions.ma>, relay=none, delay=31, delays=0.21/0.07/31/0, dsn=4.4.1, status=deferred (connect to e-solutions.ma[5.189.177.237]:25: Operation timed out)
smtp_1       | Jan 26 19:58:34 mail postfix/smtp[19532]: 5AB6AFE10F: to=<r.benaddi@e-solutions.ma>, relay=none, delay=586, delays=556/0.1/30/0, dsn=4.4.1, status=deferred (connect to e-solutions.ma[5.189.177.237]:25: Operation timed out)

**Comment by kaiyou** [comment]
The timeout at the end looks like your outgoing port 25 is filtered by some kind of firewall.

If you want outbound mail to work, you will need to get this open. Very probably your provider has it filtered, it is now a matter of finding out if they can lift the constraint after you show that you are not a spammer :)

**Comment by kaiyou** [comment]
Any news?

**Comment by Fiasee** [comment]
A simple search on the internet reveals that DO doesn't usually open port 25 on request, can we have a provision to set the outgoing port to some other port in the env file?

**Comment by Fiasee** [comment]
DO doesn't usually open port 25 on request, can we have a provision to set the outgoing port to some other port in the env file?

**Comment by furkanmustafa** [comment]
> DO doesn't usually open port 25 on request, can we have a provision to set the outgoing port to some other port in the env file?

outgoing (target) port is the port of the other party. You cannot configure that (except for specific targets)

**Comment by Fire-Dragons** [comment]
> The timeout at the end looks like your outgoing port 25 is filtered by some kind of firewall.
> 
> If you want outbound mail to work, you will need to get this open. Very probably your provider has it filtered, it is now a matter of finding out if they can lift the constraint after you show that you are not a spammer :)

have other solutions?

**Comment by Fire-Dragons** [comment]
> The timeout at the end looks like your outgoing port 25 is filtered by some kind of firewall.
> 
> If you want outbound mail to work, you will need to get this open. Very probably your provider has it filtered, it is now a matter of finding out if they can lift the constraint after you show that you are not a spammer :)

i use overrides/postfix.master to add master.cf
```
service_name/type=submission inet n       -       y      -       -       smtpd
service_name/type=smtps     inet  n       -       -       -       -       smtpd
service_name/type=submission      unix  -       -       n       -       -       smtp
service_name/type=smtps      unix  -       -       n       -       -       smtp
```
but postfix still use port 25 to send email
how do to use other port to send email?

**Comment by brunoberte** [comment]
I have a similar problem with an AWS EC2 instance. I've discovered that the port 25 is blocked by default. (https://aws.amazon.com/pt/premiumsupport/knowledge-center/ec2-port-25-throttle/)

I think we should receive an error when the server can't send the email right? like an automatic response or some notification on the admin panel.

**Comment by Nebukadneza** [comment]
Hi There,

The `Mailu`-Project is currently in a bit of a bind! We are short on man-power, and we need to judge if it is possible for us to put in some work on this issue.

To help with that, we are currently trying to find out which issues are actively keeping users from using `Mailu`, which issues have someone who want to work on them — and which issues may be less important. These a less important ones could be discarded for the time being, until the project is in a more stable and regular state once again.

In order for us to better assess this, it would be helpful if you could put a **reaction on this post** (use the :smiley: icon to the top-right).
- 👍️ if you **need this** to be able to use Mailu. Ideally, you’d also be able to test this on your installation, and provide feedback …
- 🎉 if you find it a **nice bonus**, but no deal-breaker
- 🚀 if you want to **work on it yourself**!
We want to keep this **voting open for 2 weeks** from now, so please help out!

**Comment by Diman0** [comment]
I am closing this since  there is no activity.. The root cause is most likely port 25 was blocked. This will be added to the documentation. If you still have issues, feel free to open a new issue and we will continue there.
